nodeadm/internal/daemon/interface.go

```diff
@@ -1,20 +1,21 @@
 package daemon
 
-import "github.com/awslabs/amazon-eks-ami/nodeadm/internal/api"
+import (
+	"context"
+
+	"github.com/awslabs/amazon-eks-ami/nodeadm/internal/api"
+)
 
 type Daemon interface {
 	// Configure configures the daemon.
 	Configure(*api.NodeConfig) error
-
 	// EnsureRunning ensures that the daemon is running.
 	// If the daemon is not running, it will be started.
 	// If the daemon is already running, and has been re-configured, it will be restarted.
```

Comment:
update the doc to specify that it should block until running?

Reply:
updated 👍